ansible/playbook/roles/generate_inventory/templates/cloud.inventory.j2

```diff
@@ -1,56 +1,58 @@
-# Create an OSEv3 group that contains the masters and nodes groups
 [OSEv3:children]
 masters
 nodes
 etcd
 
-# Set variables common for all OSEv3 hosts
 [OSEv3:vars]
 ansible_user=root
 
+public_ip_address = {{ ip_address }}
+host_key_checking = False
+
 # For Atomic VM
-containerized=false
+containerized = false
 
 openshift_release=v3.7
 openshift_image_tag=v3.7.0
 openshift_pkg_version="-3.7.0"
+
 openshift_deployment_type=origin
-openshift_enable_service_catalog=true
```

Comment:
Is there a reason why you removed it as it is required for the lab ? should we enable/disable base on a property passed when we generate the inventory ? @geoand

Reply:
Actually I haven't removed it, I just changed it's position in the file to match where it is in the playbook that exists under `virtualbox`. See this: https://github.com/snowdrop/cloud-native-infra/blob/365b049f557ee32f6350aac53ed8e7eb6009dbdc/ansible/playbook/roles/generate_inventory/templates/cloud.inventory.j2#L36